addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3178,6 +3194,14 @@ registry.SnippetSave = SnippetOptionWidget.extend({
             dialog.on('closed', this, () => resolve());
         });
     },
+    async _confirmReload() {
```

Comment:
```javascript
/**
 * @private
 */
```
at least ;) ... and in a private section. Or better: don't make a function for this stuff